cluster/kubemark/config-default.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Acutally there's a comment for it. We won't as long as defaults will be sane kubemark-wise.